tasks.c

```diff
@@ -1708,6 +1708,7 @@ static void prvAddNewTaskToReadyList( TCB_t * pxNewTCB )
                 mtCOVERAGE_TEST_MARKER();
             }
 
+            traceMOVED_TASK_TO_SUSPENDED_LIST(pxTCB);
```

Comment:
Is this macro needed?  A few lines earlier is the traceTASK_SUSPEND(pxTCB) macro (line 1688) So we already know that the function is called and the task is being suspended.  Is it necessary to know that the suspension is happening 23 lines of code later?

Reply:
I agree this macro is not necessarily needed.